packages/dialog/src/vaadin-dialog.d.ts

```diff
@@ -102,6 +109,7 @@ export type DialogEventMap = DialogCustomEventMap & HTMLElementEventMap;
  *
  * @fires {CustomEvent} resize - Fired when the dialog resize is finished.
  * @fires {CustomEvent} opened-changed - Fired when the `opened` property changes.
+ * @fires {CustomEvent} vaadin-overlay-closed - Fired after the overlay is closed.
```

Comment:
The description and the event name differ from how it's defined in `vaadin-dialog.js`